indico/modules/events/registration/templates/management/print_badges.html

```diff
@@ -49,21 +49,48 @@
             'use strict';
 
             var templates = {{ templates | tojson }};
+            var $template = $('#badge-settings-form #template');
             var $pageLayout = $('#page_layout');
             var $dashedBorder = $('#dashed_border');
+            var $pageSize = $('#page_size');
+            var $pageOrientation = $('#page_orientation');
             var $marginEditor = $('.margin-editor');
 
-            $('#badge-settings-form #template').on('change', function() {
+            var toggleFoldableOption = function(template, page_size, page_orientation) {
+                var foldable_pairs = {
+                    'A0': 'A2',
+                    'A1': 'A3',
+                    'A2': 'A4',
+                    'A3': 'A5',
+                    'A4': 'A6',
+                    'A5': 'A7',
+                    'A6': 'A8'
+                };
+                if(foldable_pairs[page_size] == template.format && page_orientation == template.orientation){
+                    $("#page_layout option[value='foldable']").prop('disabled', false);
+                } else {
+                    $("#page_layout option[value='foldable']").prop('disabled', true);
+                    if($('#page_layout :selected').val() == 'foldable'){
+                        $('#page_layout :selected').next().attr('selected', 'selected');
```

Comment:
What if the selected option is already the last one? Also, use `.val(valueOfTheOption)` on the select field to change the selection